feat: ✨  Add newsletter checkbox to installation process

EmilK15 opened the pull request:
Closes #892

Pavel910 commented on packages/app-security/src/admin/plugins/installation.tsx:
```diff
@@ -156,6 +171,15 @@ const Install = ({ onInstalled }) => {
+                                <Grid>
+                                    <Cell span={12}>
+                                        <Bind name="subscribed">
+                                            <Checkbox label={t`Subscribe to Newsletter`} />
```
@SvenAlHamad @EmilK15 should we add some more text here so user knows what this is?

SvenAlHamad replied:
I agree! @EmilK15 please set the message to something like this:
```
I want to receive updates on product improvements and new features. Doing so I accept Webiny's [privacy policy](https://www.webiny.com/privacy-policy/). 
```

EmilK15 commented:
Added the above comments.
Also removed the unneeded text below password input.
Added a try/catch block to fetch code so that if it does fail it will toss and error

EmilK15 commented:
![image](https://user-images.githubusercontent.com/9532882/83445940-e1590000-a41b-11ea-83bb-8b602c304e15.png)
![image](https://user-images.githubusercontent.com/9532882/83445965-e9b13b00-a41b-11ea-83d3-6804e610ee92.png)
Current iteration of changes for new Welcome page
